File: dayTwo/Javascript/After Break/QuestionTwo.js
let grade=(firstScore, secondScore, thirdScore)=>{
    let average = 0;
    let gradeFor = '';
    if(firstScore >-1 && secondScore > -1 && thirdScore > -1){
        average = (firstScore + secondScore + thirdScore) / 3;
        
        if(average >= 90 && average <= 100){
            gradeFor = 'A';
        }
        else if(average >= 80 && average < 90){
            gradeFor = 'B';
        }
        else if(average >= 70 && average < 80){
            gradeFor = 'C';
        }
        else if(average >= 60 && average < 70){
            gradeFor = 'D';
        }
        else if(average >= 0 && average < 60){
            gradeFor = 'F';
        }
    }
    
return gradeFor;
}

console.log(grade(64,78,95));

File: dayTwo/python/After Break/question_two.py

def grade(first_score,second_score,third_score):
    
    if(first_score > -1 and second_score > -1 and third_score > -1):
        average = (first_score + second_score + third_score) / 3
        
        if(average >= 90 and average <= 100):
            return 'A'
        elif(average >= 80 and average < 90):
            return  'B'
        elif(average >= 70 and average < 80):
            return  'C'
        elif(average >= 60 and average < 70):
            return  'D'
        elif(average >= 0 and average < 60):
            return  'F'
    

